book/intro.md

```diff
@@ -3,7 +3,7 @@
 
 This book helps to explore tools and datasets, that are useful for researchers working topics ranging from processes in the carbon cycle to machine learning tools and remote sensing. This book will be tested within the ["Advancing ecosystem carbon flux research"](https://www.lorentzcenter.nl/advancing-ecosystem-carbon-flux-research.html) at the Lorentz Center between 14 - 18 October 2024 and be used to inform the discussion on research priorities for the terrestrial carbon cycle and technical developments for research in the next decade. 
 
-The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. A longer overview of the material and sections covered in the workshop can be found [here](https://excited-co2.github.io/workshop_tutorial/main/sections/overview.html)
+The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. A longer overview of the material and sections covered in the workshop can be found [here](https://excited-co2.github.io/workshop_tutorial/main/overview.html)
```

Comment:
Could you change this into a relative link? I.e. just to `overview.md`. The jupyter book builder will then resolve the link correctly when the book is built.